### PR TITLE
Re-added NiM link to documentation.  Also 25 language additions to debugging documentation.

### PR DESCRIPTION
**Pull request by june07**
@nodejs/tsc #1923

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/localization ：This user has a great contributions of 25 languages. Can anyone of you related to the specific language to have a review of that?

**Comment by taggon** [comment]
The Korean translation looks good.

File: locale/ko/docs/guides/debugging-getting-started.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ info on these follows:
   are listed.
 * **Option 2**: Copy the `devtoolsFrontendUrl` from the output of `/json/list`
   (see above) or the --inspect hint text and paste into Chrome.
+* **Option 2**: Install the Chrome Extension NIM (Node Inspector Manager):
```
Comment (yous) [inline locale/ko/docs/guides/debugging-getting-started.md]: This should be "Option 3".
```suggestion
* **Option 3**: Install the Chrome Extension NIM (Node Inspector Manager):  
```

File: locale/ko/docs/guides/debugging-getting-started.md [hunk 2]
```diff
@@ -131,6 +133,8 @@ Node 인스펙터에 접속할 수 있는 여러 상용 도구와 오픈소스 
   Configure 버튼을 눌러서 대상 호스트와 포트 목록을 확인합니다.
 * **방법 2**: `/json/list`(상단 참고)의 출력에서 `devtoolsFrontendUrl`을
   복사하거나 --inspect가 알려준 텍스트에서 복사해서 크롬에 붙여넣기를 합니다.
+* **옵션 2**: 크롬 확장프로그램 NIM(Node Inspector Manager)을 설치하세요.
```
Comment (yous) [inline locale/ko/docs/guides/debugging-getting-started.md]:
```suggestion
* **방법 3**: 크롬 확장프로그램 NIM(Node Inspector Manager)을 설치하세요. 
```

**Review by yous — COMMENTED** [review]
Reviewed `ko` directory.

File: locale/nl/docs/guides/debugging-getting-started.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
+---
+title: debuggen-starten
+layout: docs.hbs
+---
+
+# Debuggids
```
Comment (AdriVanHoudt) [inline locale/nl/docs/guides/debugging-getting-started.md]: I think this should be 2 words `Debug gids`

File: locale/nl/docs/guides/debugging-getting-started.md [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+
+Deze gids zal u helpen met het debuggen van uw Node.js apps en scripts.
+
+# Inspecteur Inschakelen
+
+Wanneer gestart met de **--inspecteer** switch, een Node.js-proces luistert via WebSockets
```
Comment (AdriVanHoudt) [inline locale/nl/docs/guides/debugging-getting-started.md]: The node flag stays `--inspect`.

File: locale/nl/docs/guides/debugging-getting-started.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+voor diagnostische opdrachten zoals gedefinieerd in het [Inspector Protocol][],
+standaard bij host en poort 127.0.0.1: 9229. Elk proces krijgt ook een
+uniek [UUID] [] (bv. "0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e").
```
Comment (AdriVanHoudt) [inline locale/nl/docs/guides/debugging-getting-started.md]:
```suggestion
Wanneer Node.js gestart wordt met de **--inspect** switch, luistert het process via WebSockets
voor diagnostische opdrachten zoals gedefinieerd in het [Inspector Protocol][],
standaard op host en poort 127.0.0.1:9229. Elk proces krijgt ook een
uniek [UUID] [] (bv. "0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e").
```

**Review by ChALkeR — DISMISSED** [review]
I believe this should not be merged without input from @fhemberger, or unless there is a consensus that we should disregard the inicial concern.

There is also some active discussion going on in https://github.com/nodejs/nodejs.org/issues/1923#issuecomment-448322228 to research better options, and I believe that this should be blocked until that finishes.

**Comment by fhemberger** [comment]
Labeled "on hold" until the discussion in #1923 is resolved. Please do not merge until then.

**Comment by june07** [comment]
@fhemberger The discussion in #1923 concluded 9 days ago.  Issue was said to be more appropriately handled here.  Any advice on getting this moved forward?

File: locale/no/docs/guides/debugging-getting-started.md [hunk 6]
```diff
@@ -0,0 +1,262 @@
+---
+title: Debugging - Komme i Gang
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
title: Debugging - Hvordan komme i gang
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+layout: docs.hbs
+---
+
+# Debugging Guide
+
+Denne guiden vil hjelpe deg med å komme i gang debugging din Node.js programmer og skript.
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
Denne guiden vil hjelpe deg med å komme i gang med debugging av dine Node.js programmer og skript.
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+
+## Aktivere Inspector
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
## Aktiver Inspector
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+
+Når begynte med **--inspisere** * slå, en Node.js prosessen lytter via WebSockets
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
Når en node prosess startes med **--inspect** vil den lytte via WebSockets
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+for diagnostiske kommandoer som definert av [Inspector-Protokollen][],
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
for diagnostiske kommandoer som er definert av [Inspector-Protokollen][],
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+standard på vertsnavn og port 127.0.0.1:9229. Hver prosess er også tildelt en
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
som standard på vertsnavn og port 127.0.0.1:9229. Hver prosess er også tildelt en
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 12]
```diff
@@ -0,0 +1,262 @@
...
+unik [UUID][] (f.eks `0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e`).
+
+Inspektøren kunder må vite og angi vertsadresse, port, og UUID å koble
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
Inspector klienter må vite og angi vertsadresse, port, og UUID for å koble
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 13]
```diff
@@ -0,0 +1,262 @@
...
+til WebSocket grensesnitt. Den fullstendige URL-er
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
til WebSocket grensesnitt. Den fullstendige URL er
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 14]
```diff
@@ -0,0 +1,262 @@
...
+`ws://127.0.0.1:9229/0f2c936f-b1cd-4ac9-aab3-f63b0f33d55e`, selvfølgelig avhengig
+på faktiske vertsnavn og port, og med riktig UUID for eksempel.
+
+Inspektøren inneholder også en HTTP-endepunkt å tjene metadata om debuggee,
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
Inspektøren inneholder også et HTTP-endepunkt for å tilby metadata om prosessen som debugges,
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 15]
```diff
@@ -0,0 +1,262 @@
...
+inkludert WebSocket URL, UUID og Chrome DevTools URL. Få denne metadata
+ved å sende en HTTP-forespørsel til `http://[vert:port]/json/list". Dette gir en
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
ved å sende en HTTP-forespørsel til `http://[vert:port]/json/list". Dette gir et
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 16]
```diff
@@ -0,0 +1,262 @@
...
+JSON-objekt som følgende; bruk `webSocketDebuggerUrl` eiendom som
+URL-adressen til å koble direkte til Inspektør.
+
+<!-- eslint-hopp -->
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
<!-- eslint-skip -->
```

File: locale/no/docs/guides/debugging-getting-started.md [hunk 17]
```diff
@@ -0,0 +1,262 @@
...
+``javascript
+{
+"description": "node.js forekomst",
```
Comment (eiriksm) [inline locale/no/docs/guides/debugging-getting-started.md]:
```suggestion
"description": "node.js instans",
```

**Review by eiriksm — CHANGES_REQUESTED** [review]
Some comments on the Norwegian seemingly auto-translated content

File: locale/tr/docs/guides/debugging-getting-started.md [hunk 18]
```diff
@@ -0,0 +1,262 @@
+---
+title: hata ayıklama - Başlarken
+layout: docs.hbs
```
Comment (gokaygurcan) [inline locale/tr/docs/guides/debugging-getting-started.md]: Everything in this file needs to be reviewed by a native Turkish speaker. I want to review and fix the problems when I have some time earlier next year (no pun intended).

Same for `locale/tr/site.json`.

**Review by gokaygurcan — CHANGES_REQUESTED** [review]

File: locale/tr/docs/guides/debugging-getting-started.md (hunk 18 above)
Comment (fhemberger) [reply locale/tr/docs/guides/debugging-getting-started.md]: /cc @nodejs/nodejs-tr

**Comment by june07** [comment]
Any more native speaker assistance on the +1s for this PR
...

**Comment by fhemberger** [comment]
@nodejs/website Everybody, please review the translation for your language in this PR, so this change can land.

File: locale/ru/site.json [hunk 19]
```diff
@@ -0,0 +1,152 @@
+{
+  "title": "Node.js",
+  "author": "Node.js Foundation",
+  "url": "https://nodejs.org/en/",
+  "locale": "ru",
+  "scrollToTop": "Наверх",
+  "reportNodeIssue": "Вопрос Node.js отчет ",
```
Comment (sudodoki) [inline locale/ru/site.json]: reportNodeIssue: `Сообщить о проблеме с Node.js`
reportWebsiteIssue: `Сообщить о проблеме с сайтом`
getHelpIssue: `Помощь`
by: `от`
all-downloads: `Все варианты скачиваний `
chakracore-nightly: `Ночные сборки Node-ChakraCore`
previous: `Предыдущий`

File: locale/ru/site.json [hunk 20]
```diff
@@ -0,0 +1,152 @@
...
+      "link": "feed/releases.xml",
+      "text": "Блог Node.js : Релизы"
+    },
+    {
+      "link": "feed/vulnerability.xml",
+      "text": "Блог Node.js : Сообщения Об Уязвимостях "
+    }
+  ],
+  "home": {
+    "text": "Дома"
```
Comment (sudodoki) [inline locale/ru/site.json]: home.text: `Главная`

File: locale/ru/site.json [hunk 21]
```diff
@@ -0,0 +1,152 @@
...
+  },
+  "about": {
+    "link": "about",
+    "text": "О",
```
Comment (sudodoki) [inline locale/ru/site.json]: `О проекте`

File: locale/ru/site.json [hunk 22]
```diff
@@ -0,0 +1,152 @@
...
+      "text": "Товарный знак"
+    },
+    "privacy": {
+      "link": "about/privacy",
+      "text": "Политика Конфиденциальности "
+    }
+  },
+  "download": {
+    "link": "download",
+    "text": "Загрузок",
```
Comment (sudodoki) [inline locale/ru/site.json]: download.text: `Загрузки`

File: locale/ru/site.json [hunk 23]
```diff
@@ -0,0 +1,152 @@
...
+    "install-on-linux": {
+      "text": "Установка Node.js через бинарный архив в Linux"
+    }
+  },
+  "docs": {
+    "link": "docs",
+    "text": "Документы",
+    "es6": {
+      "link": "docs/es6",
+      "text": "ES6 и за его пределами"
```
Comment (sudodoki) [inline locale/ru/site.json]: es6.text: `ES6 и выше`

File: locale/ru/site.json [hunk 24]
```diff
@@ -0,0 +1,152 @@
...
+    },
+    "api-lts": {
+      "link": "/dist/latest-%ver-major%/docs/api",
+      "subtext": "LTS",
+      "text": "%Вер% по API"
```
Comment (sudodoki) [inline locale/ru/site.json]: I guess this interpolation won't work as it's not `%ver%` exactly? For this few I would just go with `%ver% API`

File: locale/ru/site.json [hunk 25]
```diff
@@ -0,0 +1,152 @@
...
+    },
+    "api-current": {
+      "link": "/dist/latest-%ver-major%/docs/api",
+      "text": "%Вер% по API"
+    },
+    "guides": {
+      "link": "docs/guides",
+      "text": "Гиды"
```
Comment (sudodoki) [inline locale/ru/site.json]: guides.text: `Руководство`

File: locale/ru/site.json [hunk 26]
```diff
@@ -0,0 +1,152 @@
...
+    },
+    "dependencies": {
+      "link": "docs/meta/topics/dependencies",
+      "text": "Зависимостей"
```
Comment (sudodoki) [inline locale/ru/site.json]: dependencies.text: 'Зависимости'

File: locale/ru/site.json [hunk 27]
```diff
@@ -0,0 +1,152 @@
...
+    }
+  },
+  "getinvolved": {
+    "link": "get-involved",
+    "text": "Ввязываться",
```
Comment (sudodoki) [inline locale/ru/site.json]: getinvolved.text: 'Присоедениться'

File: locale/ru/site.json [hunk 28]
```diff
@@ -0,0 +1,152 @@
...
+    "link": "blog",
+    "text": "Новости"
+  },
+  "foundation": {
+    "link": "https://foundation.nodejs.org/",
+    "text": "Фонд"
+  },
+  "releases": {
+    "title": "Release History",
+    "downloads": "Downloads"
```
Comment (sudodoki) [inline locale/ru/site.json]: releases.downloads: `Загрузки`

File: locale/ru/site.json [hunk 29]
```diff
@@ -0,0 +1,152 @@
...
+  },
+  "links": {
+    "pages": {
+      "changelog": "Changelog"
```
Comment (sudodoki) [inline locale/ru/site.json]: changelog: `Список изменений`

File: locale/ru/site.json [hunk 30]
```diff
@@ -0,0 +1,152 @@
...
+      "text": "Вклад"
+    },
+    "conduct": {
+      "link": "https://github.com/nodejs/node/blob/master/doc/guides/contributing/coc.md#code-of-conduct",
+      "text": "Кодекс поведения"
+    }
+  },
+  "security": {
+    "link": "security",
+    "text": "Безопасности"
```
Comment (sudodoki) [inline locale/ru/site.json]: security.text: `Безопасность`

File: locale/no/site.json [hunk 31]
```diff
@@ -0,0 +1,152 @@
...
+  "author": "Node.js Foundation",
+  "url": "https://nodejs.org/en/",
+  "locale": "no",
+  "scrollToTop": "Bla til toppen",
+  "reportNodeIssue": "Rapport Node.js problemet",
+  "reportWebsiteIssue": "Rapporten nettstedet problemet",
+  "getHelpIssue": "Få Hjelp",
+  "by": "av",
+  "all-downloads": "All last valg",
+  "nightly": "Nightly builds",
```
Comment (phillipj) [inline locale/no/site.json]:
```suggestion
  "nightly": "Nattlige bygg",
```

File: locale/no/site.json [hunk 32]
```diff
@@ -0,0 +1,152 @@
...
+  "title": "Node.js",
+  "author": "Node.js Foundation",
+  "url": "https://nodejs.org/en/",
+  "locale": "no",
+  "scrollToTop": "Bla til toppen",
+  "reportNodeIssue": "Rapport Node.js problemet",
+  "reportWebsiteIssue": "Rapporten nettstedet problemet",
+  "getHelpIssue": "Få Hjelp",
+  "by": "av",
+  "all-downloads": "All last valg",
```
Comment (phillipj) [inline locale/no/site.json]:
```suggestion
  "all-downloads": "Alle nedlastingsalternativer",
```

File: locale/no/site.json [hunk 33]
```diff
@@ -0,0 +1,152 @@
...
+  "nightly": "Nightly builds",
+  "chakracore-nightly": "Node-ChakraCore Nightly builds",
```
Comment (phillipj) [inline locale/no/site.json]:
```suggestion
  "chakracore-nightly": "Node-ChakraCore nattlige bygg",
```

File: locale/no/site.json [hunk 34]
```diff
@@ -0,0 +1,152 @@
+{
+  "title": "Node.js",
+  "author": "Node.js Foundation",
+  "url": "https://nodejs.org/en/",
+  "locale": "no",
+  "scrollToTop": "Bla til toppen",
+  "reportNodeIssue": "Rapport Node.js problemet",
+  "reportWebsiteIssue": "Rapporten nettstedet problemet",
```
Comment (phillipj) [inline locale/no/site.json]:
```suggestion
  "reportWebsiteIssue": "Rapporter problemer med nettsiden",
```

File: locale/no/site.json [hunk 35]
```diff
@@ -0,0 +1,152 @@
+{
+  "title": "Node.js",
+  "author": "Node.js Foundation",
+  "url": "https://nodejs.org/en/",
+  "locale": "no",
+  "scrollToTop": "Bla til toppen",
+  "reportNodeIssue": "Rapport Node.js problemet",
```
Comment (phillipj) [inline locale/no/site.json]:
```suggestion
  "reportNodeIssue": "Rapporter Node.js problemer",
```

File: locale/no/site.json [hunk 36]
```diff
@@ -0,0 +1,152 @@
...
+    },
+    "package-manager": {
+      "link": "download/package-manager",
+      "text": "Installere Node.js via pakkebehandleren"
+    },
+    "shasums": {
+      "link": "SHASUMS256.txt.asc",
+      "text": "Signert SHASUMS for slipp filer",
+      "verify-link": "https://github.com/nodejs/node#verifying-binaries",
+      "verify-text": "How to verify"
```
Comment (phillipj) [inline locale/no/site.json]:
```suggestion
      "verify-text": "Hvordan verifisere"
```

**Comment by june07** [comment]
Where are we at?  Any more native speakers willing to contribute a +1 so we can get this pushed...?

**Comment by june07** [comment]
Now I'm thinking I should have maybe separated the language commits by branch or something.  Lol.

**Comment by fhemberger** [comment]
@june07 You could still split it up. Leave the translations that have been approved and remove the ones still waiting for review. Let's get this merged.

The page can still be translated later as well.

**Comment by june07** [comment]
@fhemberger Removed 22 locales which still need +1s and left 4 as follows:
Removed bg  bn  cs  da  el  et  fi  he  hi  hr  hu  id  ka  lv  pl  pt  ro  sk  sv  th  tr  vi
Left ko nl ru ro

Should I create branches for each language instead of bunching them all into a single PR moving forward?

**Comment by ZYSzys** [comment]
> Should I create branches for each language instead of bunching them all into a single PR moving forward?

I'd prefer to create branches for each language since this makes code review easier than bunching them all into one.

**Comment by fhemberger** [comment]
> Removed bg bn cs da el et fi he hi hr hu id ka lv pl pt ro sk sv th tr vi

I'm terribly sorry, I just noticed you provided _a lot_ of translations for locales we don't even have (yet) on nodejs.org. There are discussions at the moment how to improve translations for the upcoming website redesign, but for now please stick to the locales we support right now:

ar, ca, de, en, es, fa, fr, gl, it, ja, ko, pt-br, uk, zh-cn and zh-tw

We only have members to review those. Also if a translation is not available, the website will fall back to the English version, so no worries.

**Comment by june07** [comment]
No worries.  I guess that leaves ko all by itself.
Actually, nl, no, ru have gotten feedback in this PR.  Am I removing these as well or leaving them?

I do understand what you are saying about having no members on the immediate team to review.  However as I made decisions on choosing which languages to translate several months ago at the start of this, I did so by referencing node.js working groups https://nodejs.org/en/about/working-groups/#i18n.  Might that be a potential source of help to validate?

I had started in on separating into branches/language per @ZYSzys.  Should these PRs be submitted?  I could assist with pinging someone from the applicable working groups if that helps to get them +1ed.

Let me know.

**Comment by fhemberger** [comment]
Some of those groups date back to the io.js days (~4 years ago) and are not active any longer. So at the moment, only the languages you find in `locales` are supported on the site and have (more or less active) translation contributors.

**Comment by june07** [comment]
Ok.  Removed nl, no, ru as well.

**Comment by june07** [comment]
@fhemberger This should be good to go now.

**Comment by fhemberger** [comment]
@june07 Yes, it has been a long way. Thank you for your patience!

**Comment by june07** [comment]
Thanks and thank you.

**Pull request merged**